cbits/is-valid-utf8.c

```diff
@@ -50,6 +50,7 @@ SUCH DAMAGE.
 #endif
 
 #include <MachDeps.h>
+#include "Rts.h"
```

Comment:
This seems to break on Fedora for me... booting 9.4.6 with both 9.2.8 and 9.4.5:
```
In file included from /usr/lib64/ghc-9.4.5/lib/x86_64-linux-ghc-9.4.5/rts-1.0.2/include/Rts.h:235,
                 from libraries/bytestring/cbits/is-valid-utf8.c:53:0: error:
    
/usr/lib64/ghc-9.4.5/lib/x86_64-linux-ghc-9.4.5/rts-1.0.2/include/rts/OSThreads.h:38:5: error:
     error: unknown type name ‘clockid_t’
       38 |     clockid_t timeout_clk;
          |     ^~~~~~~~~
   |
38 |     clockid_t timeout_clk;
   |     ^
```

Reply:
Looks to me like that #include should be conditional somehow.